### PR TITLE
Update `solar` and `unknown` capacity for Hungary

### PR DESCRIPTION
**Pull request by torokati44**
I have deduced that "Waste" also counts as `biomass` (but capacities are unchanged) and that `unknown` is "Other" + "Other renewable".

[[source](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show?name=&defaultValue=false&viewType=TABLE&areaType=BZN&atch=false&dateTime.dateTime=01.01.2024+00:00|UTC|YEAR&dateTime.endDateTime=01.01.2024+00:00|UTC|YEAR&area.values=CTY|10YHU-MAVIR----U!BZN|10YHU-MAVIR----U&productionType.values=B01&productionType.values=B25&productionType.values=B02&productionType.values=B03&productionType.values=B04&productionType.values=B05&productionType.values=B06&productionType.values=B07&productionType.values=B08&productionType.values=B09&productionType.values=B11&productionType.values=B12&productionType.values=B14&productionType.values=B20&productionType.values=B15&productionType.values=B16&productionType.values=B17&productionType.values=B19)]

### CODE REVIEW

File: config/zones/HU.yaml [hunk 1]
```diff
@@ -45,13 +45,19 @@ capacity:
     - datetime: '2024-01-01'
       source: entsoe.eu
       value: 3552.0
+    - datetime: '2024-08-01'
+      source: entsoe.eu
+      value: 3688.0
```
Comment (VIKTORVAV99) [inline config/zones/HU.yaml:50]: You should update the value for 2024-01-01 here instead as ENTSO-E only have yearly capacity data.

This is so it's consistent with any updates in the future.

File: config/zones/HU.yaml [hunk 2]
```diff
@@ -45,13 +45,19 @@ capacity:
...
   unknown:
     - datetime: '2023-01-01'
       source: entsoe.eu
       value: 92.0
     - datetime: '2024-01-01'
       source: entsoe.eu
       value: 97.0
+    - datetime: '2024-08-01'
+      source: entsoe.eu
+      value: 106.0
```
Comment (VIKTORVAV99) [inline config/zones/HU.yaml:60]: And same here.

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
Hi and thanks for your contribution!

Some small notes but the values themselves look good.

File: config/zones/HU.yaml (hunk 1 above)
Comment (torokati44) [reply config/zones/HU.yaml:50]: This is why I added a new entry with a new date:
image

Should I still do that?

Comment (torokati44) [reply config/zones/HU.yaml:50]: I don't mind making the change, just as another point of reference:
Screenshot_20240823-151102

Source: https://mavir.hu/documents/10258/278599663/PV+STATISZTIKA_20240801-ig_v1_HU.pdf/6e1de413-b078-4532-c1ea-8e60ddc75ecb?t=1723627018428

The current value of 3552.0 seems to be from May FWIW.

**Comment by torokati44** [comment]
I've made the date changes as requested, but IMHO this is not correct, and will be even less so in a week or so (and so on every month until next year).

**Comment by VIKTORVAV99** [comment]
@FelixDQ what is your take on this? It affect you guys more than it does the app.

**Review by FelixDQ — APPROVED** [review]
LGTM

**Review by VIKTORVAV99 — APPROVED** [review]

**Pull request merged**